kubernetes/linux/main.sh

```diff
@@ -79,7 +81,27 @@ startAMACoreAgent() {
          echo "export PA_CONFIG_PORT=$PA_CONFIG_PORT"
          echo "export CounterDataReportFrequencyInMinutes=$CounterDataReportFrequencyInMinutes"
          echo "export AMACALogFilePath=$AMACALogFilePath"
-      } >> ~/.bashrc
+      } >> ~/.bashrc  
+
+      if isOpenTelemetryLogsEnabled; then
+            echo "AMACoreAgent: Starting AMA Core Agent for Open Telemetry logs(and traces)"
```

Comment:
lets merge this log message with "AMACoreAgent: Starting AMA Core Agent since High Log scale mode is enabled" since AMACA can be enabled either OpenTelemetry and/or highlogscale enabled